### PR TITLE
feat(swag): Update Devfolio swag opportunity

### PR DESCRIPTION
**Pull request by ShivamPorwal02**
- [x] I've checked that this isn't a new swag opportunity proposal.
- [x] I've checked that this isn't a duplicate pull request.

Updated Devfolio swag opportunity.

### CODE REVIEW

**Review by aslafy-z — APPROVED** [review]

**Review by aslafy-z — CHANGES_REQUESTED** [review]
This only ships to India, please add it in the description.

**Comment by ShivamPorwal02** [comment]
@aslafy-z I have updated the description.Please review.

File: data.json [hunk 1]
```diff
@@ -155,11 +155,11 @@
   {
     "name": "Devfolio",
     "difficulty": "hard",
-    "description": "Four merged pull requests during the month of October will earn stickers and a chance at a limited edition t-shirt. Exceptional PRs will also earn an exclusive limited edition t-shirt.",
-    "reference": "https://devfolio.co/blog/hacktoberfest-2019-devfolio/",
-    "image": "https://devfolio.sgp1.digitaloceanspaces.com/ghost/2019/10/ghost_download1.jpeg",
-    "dateAdded": "2019-10-05T23:17:55.000Z",
-    "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
+    "description": "Four merged pull requests in their asked repositories during the month of October will earn swags.(Ships only to India)",
```
Comment (aslafy-z) [inline data.json:158]:
```suggestion
    "description": "Four merged pull requests in their asked repositories during the month of October will earn swags. (Ships only to India)",
```

**Review by aslafy-z — CHANGES_REQUESTED** [review]

File: data.json (hunk 1 above)
Comment (ShivamPorwal02) [reply data.json]: @aslafy-z I have updated the description, please review.

File: data.json [hunk 2]
```diff
@@ -155,11 +155,11 @@
...
     "name": "Devfolio",
     "difficulty": "hard",
-    "description": "Four merged pull requests during the month of October will earn stickers and a chance at a limited edition t-shirt. Exceptional PRs will also earn an exclusive limited edition t-shirt.",
-    "reference": "https://devfolio.co/blog/hacktoberfest-2019-devfolio/",
-    "image": "https://devfolio.sgp1.digitaloceanspaces.com/ghost/2019/10/ghost_download1.jpeg",
-    "dateAdded": "2019-10-05T23:17:55.000Z",
-    "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
+    "description": "Four merged pull requests in their asked repositories during the month of October will earn swags. (Ships only to India)",
+    "reference": "https://devfolio.co/blog/devfolio-hacktoberfest-2020/",
+    "image": "https://pbs.twimg.com/media/EJfpSDTUUAIXtdP?format=jpg&name=900x900",
```
Comment (aslafy-z) [inline data.json:160]:
```suggestion
    "image": "https://pbs.twimg.com/media/EJfpSDTUUAIXtdP.jpg",
```

**Review by aslafy-z — CHANGES_REQUESTED** [review]

File: data.json (hunk 2 above)
Comment (ShivamPorwal02) [reply data.json]: @aslafy-z I have updated image,please review.

**Review by aslafy-z — APPROVED** [review]
I fixed Deque failing check, you can rebase your branch if you want to see a green check.

**Comment by ShivamPorwal02** [comment]
@aslafy-z Okay

File: data.json [hunk 3]
```diff
@@ -155,11 +155,11 @@
   {
     "name": "Devfolio",
     "difficulty": "hard",
-    "description": "Four merged pull requests during the month of October will earn stickers and a chance at a limited edition t-shirt. Exceptional PRs will also earn an exclusive limited edition t-shirt.",
-    "reference": "https://devfolio.co/blog/hacktoberfest-2019-devfolio/",
-    "image": "https://devfolio.sgp1.digitaloceanspaces.com/ghost/2019/10/ghost_download1.jpeg",
-    "dateAdded": "2019-10-05T23:17:55.000Z",
-    "tags": ["stickers", "clothing", "hacktoberfest", "expired"]
+    "description": "Four merged pull requests in their asked repositories during the month of October will earn swags. (Ships only to India)",
```
Comment (aslafy-z) [inline data.json:158]:
```suggestion
    "description": "Contribute 4 valid PRs to one of Devfolio's approved repositories, and receive some stickers! Top 50 contributors will also get a t-shirt. (Ships only to India)",
```

**Pull request merged**